src/sentry/utils/arroyo.py

```diff
@@ -157,43 +158,36 @@ def close(self) -> None:
             self.__pool.close()
 
 
-class RunTaskWithMultiprocessing(ArroyoRunTaskWithMultiprocessing[TStrategyPayload, TResult]):
+def run_task_with_multiprocessing(
+    *,
+    pool: MultiprocessingPool,
+    function: Callable[[Message[TStrategyPayload]], TResult],
```

Comment:
splitting this parameter out is important so that the generics can be resolved correctly